pyrate/correct.py

```diff
@@ -150,18 +138,38 @@ def _update_params_with_tiles(params: dict) -> None:
     params[cf.TILES] = tiles
 
 
-process_steps = {
+correct_steps = {
     'orbfit': orb_fit_calc_wrapper,
     'refphase': ref_phase_est_wrapper,
     'mst': mst_calc_wrapper,
     'apscorrect': wrap_spatio_temporal_filter,
     'maxvar': maxvar_vcm_calc_wrapper,
-    'timeseries': timeseries_calc_wrapper,
-    'stack': stack_calc_wrapper
+    # 'timeseries': timeseries_calc_wrapper,
+    # 'stack': stack_calc_wrapper
 }
 
 
-def process_ifgs(params: dict) -> None:
+def timeseries(params: dict) -> None:
+    mpi_vs_multiprocess_logging("timeseries", params)
+    timeseries_calc_wrapper(params)
+
+
+def stack(params: dict) -> None:
+    mpi_vs_multiprocess_logging("stack", params)
+    stack_calc_wrapper(params)
+
+
+def load_params_from_disc(params: dict) -> dict:
+    params_file = Path(params[cf.OUT_DIR], 'correction.params')
+    if not params_file.exists():
+        raise FileNotFoundError("Params file not found on disc. Please run 'correct'")
+
+    with open(params_file, 'rb') as f:
+        params = cp.load(f)
+    return params
```

Comment:
This function is only called in `pyrate.main` so I suggest moving it to `main.py` and make it an internal function

Reply:
I can see this function `load_params_from_disc` has instead been deprecated